packages/password-input/src/PasswordInput/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { NotificationProps, States } from './PasswordInput.types';
+
+/**
+ * Utility function that checks if the values in an array are all equal
+ *
+ * Returns a boolean
+ *
+ * @param arr `Array<States>`
+ */
+export function allEqual(arr: Array<States>): boolean {
+  return new Set(arr).size == 1;
```

Comment:
🚀

Reply:
Nice 🚀 . Could we add a generic version of this to `lib`?